index.html

```diff
@@ -26,6 +26,7 @@
   </head>
   <body class="skin-blue sidebar-mini">
     <div id='root'>
+      Loading...
```

Comment:
Does this make sense? How long is a user usually waiting? We can use an svg loader here if we need to let the user know we are waiting.